EssentialFeed/EssentialFeedTests/Image Comments/Presentation/ImageCommentsPresenterTests.swift

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+
+class ImageCommentsPresenterTests: XCTestCase {
+	func test_title_isLocalized() {
+		XCTAssertEqual(ImageCommentsPresenter.title, localized("IMAGE_COMMENTS_VIEW_TITLE"))
+	}
+
+	func test_map_createsViewModel() {
+		let now = Date()
+		let calendar = Calendar(identifier: .gregorian)
+		let locale = Locale(identifier: "en-US")
```

Comment:
Not sure if `en-US` is a valid identifier here. I recommend using `en_US_POSIX`.

```suggestion
		let locale = Locale(identifier: "en_US_POSIX")
```